File: C#/백준/Bronze/10821. 정수의 개수/정수의 개수.cs
using System;

namespace q{
    class w{
        static void Main(){
            String a = Console.ReadLine();
            int cnt=1;
            for (int i=0;i<a.Length;i++){
                if (a[i]==','){
                    cnt++;
                }
            }
            Console.WriteLine(cnt);
        }
    }
}

File: Java/백준/Bronze/10821. 정수의 개수/정수의 개수.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String s = in.next();
        int cnt = 1;
        for (int i=0;i<s.length();i++){
            if (s.charAt(i)==','){
                cnt++;
            }
        }
        System.out.println(cnt);
    }
}